staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -950,8 +959,8 @@ func (v schemaCoercingConverter) ConvertFieldLabel(gvk schema.GroupVersionKind,
 type unstructuredSchemaCoercer struct {
 	dropInvalidMetadata bool
 
-	structuralSchema      *structuralschema.Structural
-	structuralSchemaGK    schema.GroupKind
+	structuralSchemas     map[string]*structuralschema.Structural
+	structuralSchemaGV    schema.GroupKind
```

Comment:
nit: `structuralSchemaGV` for `GroupKind` doesn't make sense?

Reply:
fixed